sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/numberFormatExpressions.scala

```diff
@@ -52,6 +52,11 @@ import org.apache.spark.unsafe.types.UTF8String
       > SELECT _FUNC_('12,454.8-', '99G999D9S');
        -12454.8
   """,
+  note = """
```

Comment:
I think this is a vague statement, better compatibility, 'works good'? This isn't telling users something actionable; I'm actually not sure what to make of it

Reply:
Updated.